server/src/main/java/org/tctalent/server/service/db/es/ElasticsearchServiceImpl.java

```diff
@@ -164,7 +181,11 @@ private BoolQueryBuilder computeFindByNumberQuery(String number) {
     boolQuery = filterOnDeletedStatus(boolQuery);
     boolQuery = filterOnSourceCountryRestrictions(boolQuery);
 
-    log.debug("Elasticsearch query:\n" + boolQuery);
+    LogBuilder.builder(log)
+        .action("ElasticsearchServiceImpl.computeFindByNumberQuery")
+        .message("Constructed Elasticsearch query:\n " + boolQuery)
+        .logDebug();
+
     return boolQuery;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 173:189, 223:239.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.

Reply:
Choosing not to make this refactoring in order tol keep the usage of LogBuilder consistent throughout the project.